### PR TITLE
chore(arcdata): Welcome arcdata v0.0.1 preview

### PR DESCRIPTION
**Pull request by swells**
chore(arcdata): Welcome arcdata v0.0.1 preview

Introduce arcdata  v0.0.1 preview. The source is housed in an external repository and not here.

---

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?
```
→ python scripts/ci/test_index.py -q
----------------------------------------------------------------------
Ran 9 tests in 103.891s


OK (skipped=2)
```

```
→  azdev linter --include-whl-extensions arcdata


Results 
=========

-  pass: faulty_help_example_parameters_rule 
-  pass: faulty_help_example_rule 
-  pass: faulty_help_type_rule 
-  pass: unrecognized_help_entry_rule 
-  pass: unrecognized_help_parameter_rule 
-  pass: expired_command_group 
-  pass: missing_group_help 
-  pass: expired_command 
-  pass: missing_command_help 
-  pass: no_ids_for_list_commands 
-  pass: bad_short_option 
-  pass: expired_option 
-  pass: expired_parameter 
-  pass: missing_parameter_help 
-  pass: no_parameter_defaults_for_update_commands 
-  pass: no_positional_parameters 
-  pass: option_length_too_long 
-  pass: option_should_not_contain_under_score 

No violations found for linter rules.

Run custom pylint rules.
Running pylint on extensions...

No violations found for custom pylint rules.
Linter: PASSED
```

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
arcdata

**Review by jiasli — APPROVED** [review]

File: src/index.json [hunk 1]
```diff
@@ -3512,6 +3512,7 @@
                     },
                     "extras": [],
                     "generator": "bdist_wheel (0.30.0)",
+                    "keywords": ["az", "sql", "arcdata", "cli", "azure", "sqlmi", "dc"],
```
Comment (kairu-ms) [inline src/index.json:3515]: Why need you update the keywords of application-insights?

Comment (swells) [reply src/index.json:3515]: @kairu-ms - These come from setup.py/dist-info/METADATA  hence needed for `./scripts/ci/test_index.py` to pass

Is the convention/ask to leave `keyword` blank for exts?

Thanks in advance.

Comment (kairu-ms) [reply src/index.json:3515]: You are changing the keywords of application-insights which is an extension owned by another team.

Comment (swells) [reply src/index.json:3515]: I see... Let us remove the keywords to dedicated values and push again. Thanks.

Comment (swells) [reply src/index.json:3515]: @kairu-ms - I removed the offending keywords as suggested. When time permits please review one more time. Thank you.

Comment (kairu-ms) [reply src/index.json:3515]: @swells The keywords are in wrong place. Your extension is arcdata, but you are adding keywords in application-insights extension.
```suggestion
```

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**